src/frontend/views/entity/Crud/index.tsx

```diff
@@ -121,7 +127,7 @@ function useEntityCrudView() {
       render: (
         <SelectionTab
           columns={{
-            fields: entityFields.data || [],
+            fields: (entityFields.data || []).filter(({ isId }) => !isId),
```

Comment:
This change is no longer needed as I refactored this logic to the `SelectionTab`

Reply:
Okay